packages/uikit-react-native/src/localization/StringSet.type.ts

```diff
@@ -268,12 +270,15 @@ export interface StringSet {
     CHANNEL_INPUT_ATTACHMENT_CAMERA_VIDEO: string;
     CHANNEL_INPUT_ATTACHMENT_PHOTO_LIBRARY: string;
     CHANNEL_INPUT_ATTACHMENT_FILES: string;
+    CHANNEL_INPUT_REPLY_PREVIEW_TITLE: (user: SendbirdUser) => string;
+    CHANNEL_INPUT_REPLY_PREVIEW_BODY: (message: SendbirdUserMessage | SendbirdFileMessage) => string;
 
     /** Channel > Message **/
     CHANNEL_MESSAGE_COPY: string;
     CHANNEL_MESSAGE_EDIT: string;
     CHANNEL_MESSAGE_SAVE: string;
     CHANNEL_MESSAGE_DELETE: string;
+    CHANNEL_MESSAGE_REPLY: (user: SendbirdUser | undefined) => string;
```

Comment:
relates to https://github.com/sendbird/sendbird-uikit-react-native/pull/110/files#r1263426512
```suggestion
    CHANNEL_MESSAGE_REPLY: string;
```